client/components/d3chart/d3-base/index.js

```diff
@@ -5,7 +5,7 @@
  */
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
-import { Component, createRef } from '@wordpress/element';
+import { Component, createRef } from 'react';
```

Comment:
Other components are using `@wordpress/element` instead of `react` for these dependencies:
https://github.com/woocommerce/wc-admin/blob/master/packages/components/src/search/index.js#L6
I think it would make sense keeping it consistent and use `@wordpress/element`.

Reply:
I think we should move away from `@wordpress` dependencies in `d3chart` so that it can be used in other non-WordPress related projects?

Reply:
We import [`@wordpress/element` in calypso,](https://github.com/Automattic/wp-calypso/blob/master/package.json#L50) and a long term goal AFAIK is to converge onto gutenberg– so I would stick with the `wordpress` packages. These are all NPM packages, so they _can_ be used in non-WP projects.